isamples_metadata/SESARTransformer.py

```diff
@@ -231,7 +231,9 @@ def _primaryLocationType(self) -> typing.AnyStr:
         return None
 
     def _materialType(self) -> typing.AnyStr:
-        return self._source_record_description()["material"]
+        if "material" in self._source_record_description():
```

Comment:
Can also do something like:
```python
return self._source_record_description().get("material", None)
```
To return a default value if key isn't in dict

Reply:
Oh nice, thanks for the suggestion.  Much more concise!